dev/devicelab/bin/tasks/flavors_test.dart

```diff
@@ -32,12 +34,14 @@ Future<void> main() async {
       );
 
       final String stderrString = stderr.toString();
-      if (!stderrString.contains('The Xcode project defines schemes: free, paid')) {
+      if (!stderrString.contains('"build/app/outputs/flutter-apk/app-bogus-release.apk" does not exist.')) {
         print(stderrString);
         return TaskResult.failure('Should not succeed with bogus flavor');
       }
+
+      return TaskResult.success(null);
     });
 
-    return TaskResult.success(null);
+    return installTestsResult;
```

Comment:
and this value gets validated by the runner? I checked out the code and looked at it, and was pretty confused.

Reply:
Great question. I looked the implementation myself and couldn't quickly figure it out either.

I want to say that validation happens, because I was able to get `Linux_android flavors_test` to fail at commit [9a022b4](https://github.com/flutter/flutter/pull/133719/commits/9a022b4776ef0414e6af984d783baeea5bcb7f15). See here: https://ci.chromium.org/raw/build/logs.chromium.org/flutter/led/andrewkolos_google.com/c5ce156363a133d985cde403ae830dabf31bc43ffaecfa9fe0cd8d6f1dc5314e/+/build.proto?server=chromium-swarm.appspot.com. The task [completed successfully](https://chromium-swarm.appspot.com/task?id=646a44d8da33d310) after I fixed the stderr check in the following commit [6503967](https://github.com/flutter/flutter/pull/133719/commits/65039674a6f8c4162250063244d307c35a221132).

Reply:
sounds good to me